src/daemon/emitter.cc

```diff
@@ -246,8 +289,19 @@ void Emitter::DoCheckCache(const Atomic<bool>& is_shutting_down) {
       return true;
     };
 
-    if (SearchDirectCache(incoming->flags(), incoming->current_dir(), &entry) &&
-        RestoreFromCache(HandledSource())) {
+    auto additional = ReadUnhandledSources(incoming);
+
+    if (!additional) {
+      LOG(ERROR) << "Can't open additional sources";
```

Comment:
What sources? It will be a lot easier to investigate if we put problematic file name and an error which happened while reading it into the message.